apps/server/src/external/ego.ts

```diff
@@ -18,11 +18,11 @@
  */
 
 import axios from 'axios';
+import { ForbiddenError, UnauthorizedError } from 'common';
```

Comment:
is `common` an npm package? looks like it used to be an internal import.
a bit confused about this change